src/database/blockQueryFactory.ts

```diff
@@ -357,13 +382,14 @@ AND b.global_slot_since_hard_fork <= CAST($2 AS INTEGER)
 AND b.global_slot_since_hard_fork < b.global_slot_since_genesis`;
 
 export const getMinMaxBlocksInSlotRangeQuery = (fork: number): string => {
-  let query = getMinMaxBlocksInSlotRangeQueryFork0;
+  let query = getMinMaxBlocksInSlotRangeQueryFork0v1;
+  if (configuration.blockDbVersion === 'v2' && fork == 0) { query = getMinMaxBlocksInSlotRangeQueryFork0v2; }
   if (fork == 1) { query = getMinMaxBlocksInSlotRangeQueryFork1; }
   return query;
 }
 
 const getEpochQueryv1 = `
-SELECT MIN(b.global_slot_since_genesis), MAX(b.global_slot_since_genesis)
+SELECT MIN(b.global_slot), MAX(b.global_slot)
```

Comment:
while global_slot and global_slot_since_genesis are the same in fork 0/v1, better to be consistent.